common/util.go

```diff
@@ -66,6 +67,7 @@ func IsDirectoryMounted(path string) bool {
 
 	// removing trailing / from the path
 	path = strings.TrimRight(path, "/")
+	MountPath = path
```

Comment:
Instead of doing this in this method, we shall do it in the command processing
there is a step where all validations are done and we start setting the logging infra.
"common.EnableMonitoring = options.MonitorOpt.EnableMon" at this line we can save the mount path as well.